ui/public/locales/ja_JP.json

```diff
@@ -1,2367 +1,3534 @@
+"label.enabling.vpn.access": "VPNアクセスを有効にしています",
+"label.end": "終了",
+"label.end.ip": "終了IPアドレス",
+"label.end.reserved.system.ip": "予約済み終了システムIPアドレス",
+"label.end.vlan": "終了VLAN",
+"label.end.vxlan": "終了VXLAN",
+"label.enddate": "日付(終了)",
+"label.endip": "終了IPアドレス",
+"label.endipv4": "IPv4終了IPアドレス",
+"label.endipv6": "IPv6終了IPアドレス",
+"label.endpoint": "エンドポイント",
+"label.endpoint.or.operation": "エンドポイントまたは操作",
+"label.endport": "終了ポート",
+"label.enter.token": "トークンの入力",
+"label.entityid": "エンティティ",
+"label.entitytype": "エンティティタイプ",
+"label.error": "エラー",
+"label.error.caught": "エラー発生",
+"label.error.code": "エラーコード",
+"label.error.file.read": "ファイルを読み取れません",
+"label.error.file.upload": "ファイルアップロード失敗しました",
+"label.error.rules.file.import": "有効なCSVファイルを選択してください",
+"label.error.setting": "エラー設定",
+"label.error.something.went.wrong.please.correct.the.following": "問題が発生しました。次の内容を修正してください",
+"label.error.upper": "エラー",
+"label.error.volume.upload": "ファイルを選択してください",
+"label.espencryption": "ESP暗号化",
+"label.esphash": "ESPハッシュ",
+"label.esplifetime": "ESP有効期間(秒)",
+"label.esppolicy": "ESPポリシー",
+"label.esx.host": "ESX/ESXiホスト",
+"label.event": "イベント",
+"label.event.archived": "イベントがアーカイブされました",
+"label.event.deleted": "イベントが削除されました",
+"label.event.timeline": "イベントタイムライン",
+"label.events": "イベント",
+"label.every": "毎",
+"label.example": "例",
+"label.example.plugin": "ExamplePlugin",
+"label.existingnetworks": "既存のネットワーク",
+"label.expunge": "抹消",
+"label.expunged": "抹消された",
+"label.expunging": "抹消中",
+"label.external.link": "外部リンク",
+"label.externalid": "外部ID",
+"label.externalloadbalanceripaddress": "外部ロードバランサーIPアドレス",
+"label.extra": "追加の引数",
 "label.f5": "F5",
-"label.f5.details": "F5 \u306e\u8a73\u7d30",
-"label.failed": "\u5931\u6557",
-"label.featured": "\u304a\u3059\u3059\u3081",
-"label.fetch.latest": "\u6700\u65b0\u60c5\u5831\u306e\u53d6\u5f97",
-"label.filterby": "\u30d5\u30a3\u30eb\u30bf\u30fc",
-"label.fingerprint": "\u30d5\u30a3\u30f3\u30ac\u30fc\u30d7\u30ea\u30f3\u30c8",
-"label.firewall": "\u30d5\u30a1\u30a4\u30a2\u30a6\u30a9\u30fc\u30eb",
-"label.firstname": "\u540d",
-"label.firstname.lower": "\u540d",
-"label.forceencap": "Force UDP Encapsulation of ESP Packets",
-"label.format": "\u5f62\u5f0f",
-"label.friday": "\u91d1\u66dc\u65e5",
-"label.full": "\u5b8c\u5168",
-"label.full.path": "\u30d5\u30eb \u30d1\u30b9",
+"label.f5.details": "F5の詳細",
+"label.f5.ip.loadbalancer": "F5 Big Ipロードバランサー",
+"label.failed": "失敗",
+"label.featured": "おすすめ",
+"label.fetch.latest": "最新情報の取得",
+"label.files": "Alternate Files to Retrieve",
+"label.filter": "フィルター",
+"label.filter.annotations.all": "すべてのコメント",
+"label.filter.annotations.self": "自身の作った",
+"label.filterby": "フィルター",
+"label.fingerprint": "フィンガープリント",
+"label.firewall": "ファイアウォール",
+"label.firewallrule": "ファイアウォールルール",
+"label.firstname": "名",
+"label.firstname.lower": "名",
+"label.fix.errors": "エラーを修正",
+"label.fixed": "修正されたオファリング",
+"label.for": "for",
+"label.forbidden": "Forbidden",
```

Comment:
```suggestion
"label.forbidden": "禁断",
```

Reply:
The word "forbidden" is so well known that I decided not to translate it.